pkg/services/featuremgmt/registry.go

```diff
@@ -224,15 +224,15 @@ var (
 			Name:              "unifiedRequestLog",
 			Description:       "Writes error logs to the request logger",
 			Stage:             FeatureStageGeneralAvailability,
-			Owner:             grafanaBackendGroup,
+			Owner:             grafanaBackendServicesSquad,
```

Comment:
the original agreement was that logging would be a group-wide shared responsibility - can we leave this for now?

Reply:
changed